src/wrap.jl

```diff
@@ -428,40 +428,40 @@ end
 # enum sp_return sp_set_config_rts(struct sp_port_config *config, enum sp_rts rts);
 function sp_set_config_rts(config::Config, rts::SPrts)
     ret = ccall((:sp_set_config_rts, libserialport), SPReturn,
-                (Config, SPrts), config, SPrts(rts))
+                (Config, Int32), config, Int(rts))
     handle_error(ret, loc())
     ret
 end
 
 # enum sp_return sp_set_cts(struct sp_port *port, enum sp_cts cts);
 function sp_set_cts(port::Port, cts::SPcts)
     ret = ccall((:sp_set_cts, libserialport), SPReturn,
-                (Port, SPcts), port, cts)
+                (Port, Int32), port, Int(cts))
     handle_error(ret, loc())
     ret
 end
 
 # enum sp_return sp_get_config_cts(const struct sp_port_config *config, enum sp_cts *cts_ptr);
 function sp_get_config_cts(config::Config)
-    cts = Ref{SPcts}()
+    cts = Ref{Int32}()
     ret = ccall((:sp_get_config_cts, libserialport), SPReturn,
-                (Config, Ref{SPcts}), config, cts)
+                (Config, Ref{Int32}), config, cts)
     handle_error(ret, loc())
     cts[]
 end
 
 # enum sp_return sp_set_config_cts(struct sp_port_config *config, enum sp_cts cts);
 function sp_set_config_cts(config::Config, cts::SPcts)
     ret = ccall((:sp_set_config_cts, libserialport), SPReturn,
-                (Config, SPcts), config, SPcts(cts))
+                (Config, Int32), config, Int(cts))
     handle_error(ret, loc())
     ret
 end
 
 # enum sp_return sp_set_dtr(struct sp_port *port, enum sp_dtr dtr);
 function sp_set_dtr(port::Port, dtr::SPdtr)
     ret = ccall((:sp_set_dtr, libserialport), SPReturn,
-                (Port, SPdtr), port, dtr)
+                (Port, Int32), port, Int(dtr))
```

Comment:
same here...

Should it be Int32, Int8, Int16, Int64...

I'm pretty sure that we should be more "generic", getting underlying type of SPdtr at compile time.


I also did:

    Int(dtr)

but I should have done

    Integer(dtr)

For your information

```
julia> typeof(Int(SP_DTR_ON))
Int64

julia> typeof(Integer(SP_DTR_ON))
Int32
```